pages/entity/_type/_.vue

```diff
@@ -172,9 +173,9 @@
         };
       },
       title() {
-        if (!this.entity) return this.$t('entity');
-        if (this.editorialTitle) return this.editorialTitle;
-        return this.entity.prefLabel[this.$store.state.i18n.locale];
+        if (!this.entity) return this.titleFallback(this.$t('entity'));
+        if (this.editorialTitle) return this.titleFallback(this.editorialTitle);
```

Comment:
This may be in the user's UI language or may be in English, but we do not know which from the Contentful response, so should not have the lang attribute.